api/data_explorer/dataset_faceted_search.py

```diff
@@ -65,12 +65,12 @@ class DatasetFacetedSearch(FacetedSearch):
     DatasetFacetedSearch object.
     """
 
-    def __init__(self):
+    def __init__(self, filters={}):
```

Comment:
Add pydoc on filters format. Is filter format that Elasticsearch expects documented anywhere?

Reply:
Done. The expected format is only documented by example in the elasticsearch-dsl docs as far as I can see.